README.md

```diff
@@ -63,11 +72,16 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   roslaunch godel_irb2400_support irb2400_blending_server.launch namespace:=[robot namespace]
   ```
   Client:
-  Cd into the 'src' directory of your catkin workspace and run the following:
+  Run following program:
```

Comment:
There is a number of commits with the same message, could you try and squash them into a single commit message? See [here](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) for details on how to do that